torch/quantization/quantize.py

```diff
@@ -429,7 +451,10 @@ def _convert(module, mapping=None, inplace=False):
                          nniqat.ConvBnReLU2d)
 
     for name, mod in module.named_children():
-        if type(mod) not in SWAPPABLE_MODULES:
+        # both swappable modules and observed custom modules are
+        # swapped as one unit
+        if type(mod) not in SWAPPABLE_MODULES and \
```

Comment:
nit: will the `SWAPPABLE_MODULES` be a globa variable, or are we planning on replacing it with a getter function?

Reply:
yeah we can do that when there is a need